Ska/ftp/ftp.py

```diff
@@ -81,7 +81,10 @@ def __del__(self):
         """
         Delete object
         """
-        self.ftp.close()
+        try:
```

Comment:
Looks good, but put in some detailed comments on why this is being done like so (capturing some of the PR history and what happens otherwise like mysterious/inconsistent test failures in kadi).  This problem always catches me by surprise.

Reply:
And I can confirm that the kadi test failure I documented (https://github.com/sot/ska_testr/pull/20#issuecomment-547212021) goes away.